client/src/test/stress.test.ts

```diff
@@ -0,0 +1,90 @@
+import * as assert from 'assert';
+import TestHelper, { CARBON, SETUP_TIMEOUT, SILICON, SIMPLE } from './TestHelper';
+
+suite('ViperIDE Stress Tests', () => {
+
+    suiteSetup(async function() {
+        this.timeout(SETUP_TIMEOUT);
+        await TestHelper.setup();
+        await TestHelper.startExtension();
+    });
+
+    suiteTeardown(async function() {
+        await TestHelper.teardown();
+    });
+
+    test("1. multiple fast verification requests", async function() {
+        this.timeout(11000);
```

Comment:
Why this constant? If it's actually important, pleas eadd a comment. Otherwise, consider rounding it.